src/job_manager/services.py

```diff
@@ -571,71 +571,74 @@ def map_jobs_to_gantt(self) -> list:
             raise PermissionDenied()
 
         job_data = []
-        jobs = Job.objects.prefetch_related("tasks")
+        jobs = Job.objects.all().order_by("priority")
```

Comment:
Let's keep `prefetch_related` and just add `order_by` after:

`Job.objects.prefetch_related("tasks").order_by("priority")`